cvat/apps/engine/serializers.py

```diff
@@ -362,6 +363,41 @@ def _set_write_once_fields(self, extra_kwargs):
 
         return extra_kwargs
 
+
+class JobFiles(serializers.ListField):
+    """
+    Read JobFileMapping docs for more info.
+    """
+
+    def __init__(self, *args, **kwargs):
+        kwargs.setdefault('child', serializers.CharField(allow_blank=False, max_length=1024))
+        kwargs.setdefault('allow_empty', False)
+        super().__init__(*args, **kwargs)
```

Comment:
Yes, it can be inlined. The class is introduced to encapsulate the logic behind parameters. I'd do it to most of the other complex fields too, but it's quite a big change.

Reply:
> The class is introduced to encapsulate the logic behind parameters.

What do you mean?